### ISSUE TITLE
In multisite setup theme fonts URLs render wrong

### ISSUE THREAD

**Issue by narinderbisht** [ampproject/amp-wp#1797]
I am converting my theme AMP compatible. Fonts URLs are working fine with single sites. But in multisite same theme fonts, URLs are wrong. It is showing fonts not found.

**Comment by westonruter** [comment]
Please don't open duplicate support threads on both GitHub and the support forum: https://wordpress.org/support/topic/on-multisite-theme-css-relative-fonts-urls-not-rendering-correct/

> Is that error message given by the browser? What is the theme font URL in question? Is there a `link` element in the page for loading the font?

**Comment by westonruter** [comment]
From the support topic:

>Here a screenshot of the error message. https://goo.gl/ZqheVG
> My theme has a local fonts folder, CSS embed these fonts as relative URLs. When AMP plugin converts them to amp style tag. Fonts URLs convert wrong.
> As I told it just in multi-site case.
> 
> Here my test URL: http://grexmo.com/st/writee/free/?amp
> 
> Please check in google chrome console panel.

In the CSS I can see this generated:

```css
@font-face {
    font-family: "FontAwesome";
    src: url("http://writee/free/wp-content/themes/writee-amp/assets/fonts/fontawesome-webfont.eot");
    src: url("http://writee/free/wp-content/themes/writee-amp/assets/fonts/fontawesome-webfont.eot") format("embedded-opentype"), 
        url("http://writee/free/wp-content/themes/writee-amp/assets/fonts/fontawesome-webfont.woff2") format("woff2"), 
        url("http://writee/free/wp-content/themes/writee-amp/assets/fonts/fontawesome-webfont.woff") format("woff"), 
        url("http://writee/free/wp-content/themes/writee-amp/assets/fonts/fontawesome-webfont.ttf") format("truetype"), 
        url("http://writee/free/wp-content/themes/writee-amp/assets/fonts/fontawesome-webfont.svg") format("svg");
    font-weight: normal;
    font-style: normal
}
```

The URLs are indeed invalid.

The CSS is coming from http://grexmo.com/st/writee/free/wp-content/themes/writee-amp/assets/css/style-ltr.css?ver=4.9.9:

```css
/*!
 *  Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
 *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
 */
/* FONT PATH
 * -------------------------- */
@font-face {
  font-family: 'FontAwesome';
  src: url("../fonts/fontawesome-webfont.eot");
  src: url("../fonts/fontawesome-webfont.eot") format("embedded-opentype"), url("../fonts/fontawesome-webfont.woff2") format("woff2"), url("../fonts/fontawesome-webfont.woff") format("woff"), url("../fonts/fontawesome-webfont.ttf") format("truetype"), url("../fonts/fontawesome-webfont.svg") format("svg");
  font-weight: normal;
  font-style: normal; }
```

So this is a bug.

Nevertheless, why are you not enqueueing the stylesheet straight from fontawesome.com? For example https://fontawesome.com/start but using `wp_enqueue_style()` in PHP. AMP whitelists stylesheets from fontawesome.com, so you can enqueue them freely: https://www.ampproject.org/docs/design/responsive/custom_fonts#1.-using-%3Clink%3E

**Comment by narinderbisht** [comment]
Thank you @westonruter ,
I will try to update my theme code.

**Comment by westonruter** [comment]
@narinderbisht I believe I may have fixed this issue in a PR I opened to fix another. Would you please try this alpha build of the plugin linked here: https://github.com/ampproject/amp-wp/pull/1852#issuecomment-460475798

**Comment by westonruter** [comment]
Similar issue was reported at https://wordpress.org/support/topic/css-font-face-with-incorrect-absolute-url-srcurlhttps-wp-content-themes/

The fix in #1852 resolved the problem.
